service/Cargo.toml

```diff
@@ -25,7 +25,7 @@ tracing = { workspace = true, optional = true }
 
 [dependencies.shuttle-common]
 workspace = true
-features = ["service"]
+features = ["service", "tracing", "display"]
```

Comment:
Maybe these are no longer needed :man_shrugging:

Reply:
I don't think we do, but removing it will cause quite a few compiler errors due to missing `cfg`'s in common, I will look into it.

Reply:
This was quite convoluted, will do it last or open a follow-up ticket.

Reply:
Created a ticket for this in linear.